src/main/java/net/sf/jabref/external/DroppedFileHandler.java

```diff
@@ -305,7 +306,7 @@ private boolean tryXmpImport(String fileName, ExternalFileType fileType, NamedCo
     //
     // @return true if user pushed "OK", false otherwise
     //
-    private boolean showLinkMoveCopyRenameDialog(String linkFileName, ExternalFileType fileType,
+    private boolean showLinkMoveCopyRenameDialog(String linkFileName, Optional<ExternalFileType> fileType,
```

Comment:
Only called with an ExternalFileType obtained from hardcoded "pdf", but I added a test that we have a valid type.